components/NavBar/index.js

```diff
@@ -0,0 +1,36 @@
+import Link from 'next/link';
+import React from 'react';
+import styles from './navbar.module.css';
+const NavBar = ({ personData: { photo } }) => {
+  return (
+    <div className={styles.homeHeader}>
+      <div className={styles.navBar}>
+        <img
+          className={styles.logo}
+          src="https://staging-members-rds.herokuapp.com/images/Real-Dev-Squad@1x.png"
```

Comment:
NIT: since this url is static, we can store it in a const at top and then use the reference on `src`
Let me know if this makes sense , rest all is good to go :)

Reply:
haan makes sense